docs/source/tutorials/creating_meshtype/creating_3d_tube.rst

```diff
@@ -0,0 +1,236 @@
+.. _tube:
+
+Creating a 3D tube
+====================
+
+.. role:: pyth(code)
+  :language: python
+
+Once you have create a :ref:`single-element cube <cube>`, let's try to make a 3D tube with more than 1 element.
+
+Set-up files and modules
+-------------------------
+Create a file and name it :file:`meshtype_3d_tube2.py` in
+the :file:`.\\src\\scaffoldmaker\\meshtypes` folder. Remember to load it in :file:`.\\src\\scaffoldmaker\\scaffolds.py`. Then :pyth:`import` the following
+modules which help us to create field, nodes, elements and interpolation functions.
+
+.. code-block:: python
+
+    from __future__ import division
+    import math
+    from opencmiss.utils.zinc.field import findOrCreateFieldCoordinates
+    from opencmiss.zinc.element import Element, Elementbasis
+    from opencmiss.zinc.field import Field
+    from opencmiss.zinc.node import Node
+    from scaffoldmaker.meshtypes.scaffold_base import Scaffold_base
+    from scaffoldmaker.utils.eftfactory_tricubichermite import eftfactory_tricubichermite
+
+Creating user interface options
+-------------------------------
+
+As mentioned, this time we will let the user decide the number of elements in each direction.
+Therefore we should set-up the user interface options as followed:
+
+.. code-block:: python
+
+    ...
+    class MeshType_3d_tube3(Scaffold_base):
+        @staticmethod
+        def getName():
+            return '3D Tube 3'
+
+        @staticmethod
+        def getDefaultOptions(parameterSetName='Default'):
+            return {
+                'Number of elements around' : 4,
+                'Number of elements along' : 2,
+                'Number of elements through wall' : 1,
+                'Wall thickness' : 0.25,
+                'Inside radius' : 0.25,
+                'Use cross derivatives' : False,
+            }
+
+        @staticmethod
+        def getOrderedOptionNames():
+            return [
+                'Number of elements around',
+                'Number of elements along',
+                'Number of elements through wall',
+                'Wall thickness',
+                'Tube length',
+                'Use cross derivatives',
+            ]
+
+        @staticmethod
+        def checkOptions(options):
+            for key in [
+                'Number of elements along',
+                'Number of elements through wall']:
+                if options[key] < 1:
+                    options[key] = 1
+            if (options['Number of elements around'] < 3) :
+                options['Number of elements around'] = 3
+            if (options['Wall thickness'] < 0.0) :
+                options['Wall thickness'] = 0.0
+            elif (options['Wall thickness'] > 0.5) :
+                options['Wall thickness'] = 0.5
+
+Everything is pretty similar to what we have done in the :ref:`3D cube <cube>` tutorial. Feel free to add your own options or constraints.
```

Comment:
remove the word "pretty"
Best if we don't add any extra adjectives as they are not typically used in scientific writing